lib/eslint.js

```diff
@@ -608,9 +608,11 @@ module.exports = (function() {
      * @param {Object} config An object whose keys specify the rules to use.
      * @param {(string|Object)} [filenameOrOptions] The optional filename of the file being checked.
      *      If this is not set, the filename will default to '<input>' in the rule context. If
-     *      an object, then it has "filename" and "saveState" properties.
+     *      an object, then it has "filename", "saveState", and "allowInlineConfig" properties.
      * @param {boolean} [saveState] Indicates if the state from the last run should be saved.
      *      Mostly useful for testing purposes.
+     * @param {boolean} [allowInlineConfig] Allow/disallow inline comments' ability to change config once it is set. Defaults to true if not supplied.
```

Comment:
`[filenameOrOptions.allowInlineConfig]` (since it doesn't exist as a named argument)